### PR TITLE
feat(debt): Debt-controller should delete user resources over 7 days in arrears 

### PR DESCRIPTION
**Pull request by xiao-jay**
1、除了发送通知其他都完成
2、新增e2e测试
``` mermaid
stateDiagram-v2
[*] --> 正常
正常 --> 轻度欠费: 使用资源
轻度欠费 --> 正常: 充值
note right of 轻度欠费
发送欠费通知
通过webhook拦截这个用户所有creat和 update 请求
end note
轻度欠费 --> 重度欠费: 继续使用 N
note right of 重度欠费
发送欠费通知
删除NS下所有资源
end note
重度欠费 --> 正常: 充值
```

### CODE REVIEW

File: controllers/account/api/v1/debt_types.go [hunk 1]
```diff
@@ -27,18 +27,17 @@ const (
 	DebtStatusLarge  DebtStatusType = "Large"
 	DebtPrefix                      = "debt-"
 	NormalPrice                     = 0
-	// SmallBlockWaitSecond 3 days
-	SmallBlockWaitSecond = 3 * 24 * 60 * 60
-	// MediumBlockWaitSecond 4 days
-	MediumBlockWaitSecond = 4 * 24 * 60 * 60
+	// SmallBlockWaitSecond 7 days
+	SmallBlockWaitSecond = 7 * 24 * 60 * 60
```
Comment (zzjin) [inline controllers/account/api/v1/debt_types.go:31]: Is the `BlockWaitSecond` here be the only location to control delete timeout? Witch measn if we wangto change it to 30days only need to modify here one place.

Comment (xiao-jay) [reply controllers/account/api/v1/debt_types.go:31]: no,I have  answered in the next comment.

File: controllers/account/controllers/debt_controller.go [hunk 2]
```diff
@@ -193,76 +212,63 @@ func (r *DebtReconciler) sendSmallNotice(ctx context.Context, accountName string
...
 func (r *DebtReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	const controllerName = "DebtController"
 	r.Logger = ctrl.Log.WithName(controllerName)
+	var smallBlockWaitSecondString string
+	if smallBlockWaitSecondString = os.Getenv("SMALL_BLOCK_WAIT_SECOND"); smallBlockWaitSecondString != "" {
+		smallBlockWaitSecond, err := strconv.Atoi(smallBlockWaitSecondString)
+		if err != nil {
+			r.Logger.Error(err, "SMALL_BLOCK_WAIT_SECOND is not a number")
+		}
+		DebtConfig["Small"] = int64(smallBlockWaitSecond)
```
Comment (xiao-jay) [inline controllers/account/controllers/debt_controller.go:260]: smallBlockWaitSecondString will get from env `"SMALL_BLOCK_WAIT_SECOND"`,env will get from 
```
        envFrom:
          - configMapRef:
              name: account-manager-configmap
```

File: controllers/common/metering/api/v1/extensionresourceprice_types.go [hunk 3]
```diff
@@ -26,8 +26,15 @@ const ExtensionResourcePricePrefix = "extensionresourceprice-"
...
 // ExtensionResourcePriceSpec defines the desired state of ExtensionResourcePrice
 type ExtensionResourcePriceSpec struct {
-	ResourceName string                            `json:"resourceName,omitempty"`
-	Resources    map[v1.ResourceName]ResourcePrice `json:"resources,omitempty"`
+	ResourceName      string                            `json:"resourceName,omitempty"`
+	Resources         map[v1.ResourceName]ResourcePrice `json:"resources,omitempty"`
+	GroupVersionKinds []GroupVersionKind                `json:"groupVersionKinds"`
+}
+
+type GroupVersionKind struct {
```
Comment (zzjin) [inline controllers/common/metering/api/v1/extensionresourceprice_types.go:34]: Change Custom GVK to `k8s.io/apimachinery/pkg/runtime/schema.GroupVersionKind` ?

Comment (xiao-jay) [reply controllers/common/metering/api/v1/extensionresourceprice_types.go:34]: It not have `json` tag, can't exec make install.

File: controllers/account/go.mod [hunk 4]
```diff
@@ -5,41 +5,32 @@ go 1.20
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/labring/sealos v0.0.0
-	github.com/labring/sealos/controllers/infra v0.0.0
+	github.com/labring/sealos/controllers/common v0.0.0-20230329143406-e5a204154f23
```
Comment (zzjin) [inline controllers/account/go.mod:8]: Manual change require to `github.com/labring/sealos/controllers/common v0.0.0`, do not lock to spec version (cannot lock in fact.)

Comment (xiao-jay) [reply controllers/account/go.mod]: done
```
	github.com/labring/sealos v0.0.0
	github.com/labring/sealos/controllers/common v0.0.0
	github.com/labring/sealos/controllers/metering v0.0.0
	github.com/labring/sealos/controllers/user v0.0.0
replace (
	github.com/labring/sealos => ../../
	github.com/labring/sealos/controllers/common => ../../controllers/common
	github.com/labring/sealos/controllers/metering => ../../controllers/metering
	github.com/labring/sealos/controllers/user => ../../controllers/user
)
```

**Review by zzjin — APPROVED** [review]

**Pull request merged**